### ISSUE TITLE
Reviewer 3 Comment 3

### ISSUE THREAD

**Issue by ramess101** [ramess101/JCED_FOMMS_Manuscript#32]
@mrshirts @jpotoff @msoroush 

> For polar molecules, do the authors anticipate that the GCMC-MBAR
approach will be capable of parametrizing the electrostatic charges with
a similar efficiency as it can be done for the non-polar interactions?

The GCMC-MBAR method is applicable to parameterizing electrostatic charges, although it is unknown if GCMC-MBAR will be as reliable over a wide range of charge values. From an optimization standpoint, fitting charges and van der Waals parameters simultaneously can lead to a multi-modal problem which might require MBAR to explore very large regions of parameter space to hop between local optima. I am not sure we need to discuss the optimization aspect, but maybe we should just leave electrostatics as future work.

**Comment by jpotoff** [comment]
@ramess101 If MBAR could do it in principle, it would be fine to say it can, but evaluating the efficiency would require an entirely new study (new water model anyone?) that is beyond the scope of this work.

**Comment by ramess101** [comment]
@jpotoff 

I agree. But no, I am not brave/crazy enough to take on developing a new water model

**Comment by jpotoff** [comment]
> I agree. But no, I am not brave/crazy enough to take on developing a new water model

LOL.

**Comment by mrshirts** [comment]
> I am not sure we need to discuss the optimization aspect, but maybe we should just leave electrostatics as future work.

It can be phrased more generally.  If there is overlap, it will work.  MBAR works locally very well, doesn't extend globally because of lack of overlap in phase space.  (We are working on how to use the MBAR estimates to extrapolate further in a systematic way - but that's beyond this paper, and shouldn't even be mentioned (too hypothetical at this point).

**Comment by ramess101** [comment]
@mrshirts @jpotoff @msoroush 

Not entirely satisfied with this response. Suggestions welcomed.

![image](https://user-images.githubusercontent.com/23408516/53783473-e48ad680-3ece-11e9-9759-c52d10cb8761.png)

**Comment by ramess101** [comment]
@mrshirts @jpotoff @msoroush 

I decided to just include a short paragraph at the end of the discussion section to address fitting electrostatic charges. Here is my modified response:

![image](https://user-images.githubusercontent.com/23408516/53910868-2c664680-4012-11e9-86e8-591b446ab341.png)

**Comment by mrshirts** [comment]
You could cite https://pubs.acs.org/doi/abs/10.1021/acs.jctc.5b00869 as an example of using MBAR for electrostatics and vdW simultaneously.

**Comment by ramess101** [comment]
@mrshirts 

Perfect!

**Comment by ramess101** [comment]
@mrshirts 

Were you OK with the wording for this response?

![image](https://user-images.githubusercontent.com/23408516/53977837-21212280-40c7-11e9-98e9-1516be03f8a1.png)
